<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Capture</class>
 <widget class="QMainWindow" name="Capture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>choose NIC</string>
        </property>
        <property name="buddy">
         <cstring>NIChooser</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="NIChooser">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="Ctime">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>400</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>&amp;save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>&amp;exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="CaptureButton">
        <property name="text">
         <string>begin</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DetailButton">
        <property name="text">
         <string>show &amp;detail</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>280</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>AR PL UKai TW</family>
          <underline>true</underline>
         </font>
        </property>
        <property name="text">
         <string>Copyright@ 简夏 &amp; 杰宝</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="SearchButton">
        <property name="text">
         <string>search</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="RepacketButton">
        <property name="text">
         <string>&amp;IpRecombine</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="FilterButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>&amp;Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QTableWidget" name="MainTable">
        <property name="accessibleName">
         <string/>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOn</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="horizontalScrollMode">
         <enum>QAbstractItemView::ScrollPerItem</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="columnCount">
         <number>2</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>no</string>
         </property>
         <property name="background">
          <color>
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Packet Captured</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="DetailViewer">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="styleSheet">
    <string notr="true">Capture{background-color:rgb(0, 170, 0);}</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FilterButton</tabstop>
  <tabstop>exitButton</tabstop>
  <tabstop>NIChooser</tabstop>
  <tabstop>CaptureButton</tabstop>
  <tabstop>RepacketButton</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>MainTable</tabstop>
  <tabstop>DetailViewer</tabstop>
  <tabstop>SearchButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CaptureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>NIChooser</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CaptureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>FilterButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>1231</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>Capture</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1231</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>973</x>
     <y>-36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CaptureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>SearchButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>998</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CaptureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>RepacketButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>1070</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CaptureButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>1080</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
